src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -773,6 +787,150 @@ tests:
               - name: "SceneID"
                 value: 0x01
 
+    - label:
+          "Step 8f: TH sends a AddScene command to DUT with the GroupID field
+          set to G1, the SceneID field set to 0x01, the TransitionTime field set
+          to 1000 (1s) and extension field sets holding an invalid
+          ExtensionField (Invalid attribute ID for given cluster). This should
+          fail and return a status of 0x85 (INVALID_COMMAND)."
+      PICS: S.S.C00.Rsp && PICS_SDK_CI_ONLY
+      command: "AddScene"
+      arguments:
+          values:
+              - name: "GroupID"
+                value: G1
+              - name: "SceneID"
+                value: 0x01
+              - name: "TransitionTime"
+                value: 1000
+              - name: "SceneName"
+                value: "Scene1"
+              - name: "ExtensionFieldSets"
+                value:
+                    [
+                        {
+                            ClusterID: 0x0006,
+                            AttributeValueList:
+                                [{ AttributeID: 0xFFFF, AttributeValue: 0x01 }],
```

Comment:
The spec does not have `AttributeValue`.  What am I missing?

Or rather, it talks about AttributeValue in various places, but  AttributeValuePairStruct does not have an AttributeValue field.

Reply:
I will open another PR to address that before we merge this one as we need a spec clarification.

Reply:
Keeping this open as we should not merge this before:
A: Closing the Spec PR
B: Refactoring AttributeValue to the spec description.